supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -219,7 +212,7 @@ class JesAsyncBackendJobExecutionActor(val jesParams: JesAsyncExecutionActorPara
   }
 
   private def generateJesGlobFileOutputs(wdlFile: WdlGlobFile): List[JesFileOutput] = {
-    val globName = callEngineFunctions.globName(wdlFile.value)
+    val globName = backendEngineFunctions.globName(wdlFile.value)
     val globDirectory = globName + "/"
     val globListFile = globName + ".list"
```

Comment:
These two (`globDirectory` and `globFileList`) are also in `backendEngineFunctions` too